keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/ApplicationController.kt

```diff
@@ -73,7 +75,7 @@ class ApplicationController(
       }
       entities.forEach { entity ->
         results[entity] = when (entity) {
-          "resources" -> applicationService.getResourceSummariesFor(application)
+          "resources" -> resourceHistoryService.getResourceSummariesFor(application)
```

Comment:
I still don't love this but it's fine

Reply:
I think I may have misread your previous comments on this. Will add the wrapper function back in `ApplicationService`.